konkong/rke/cluster.yml

```diff
@@ -0,0 +1,163 @@
+---
+# If you intened to deploy Kubernetes in an air-gapped environment,
+# please consult the documentation on how to configure custom RKE images.
+nodes:
+- address: konkong01.ls.lsst.org
+  hostname_override: konkong01
+  user: rke
+  role:
+  - controlplane
+  - worker
+  - etcd
+  labels:
+    role: storage-node
+    jupyterlab: ok
+- address: konkong02.ls.lsst.org
+  hostname_override: konkong02
+  user: rke
+  role:
+  - controlplane
+  - worker
+  - etcd
+  labels:
+    role: storage-node
+    jupyterlab: ok
+- address: konkong03.ls.lsst.org
+  hostname_override: konkong03
+  user: rke
+  role:
+  - controlplane
+  - worker
+  - etcd
+  labels:
+    role: storage-node
+    jupyterlab: ok
+#- address: konkong04.ls.lsst.org
+#  hostname_override: konkong04
+#  user: rke
```

Comment:
Remember to use only 3 etcd nodes

Reply:
yes, will be adjusted when all 5 nodes are ready (to only 3etcd, 5 is too overkill for a 5 node cluster)

Reply:
There's no reason **not** to use 5 etcd nodes for a 5 node cluster.